__tests__/utils/withVariables.spec.ts

```diff
@@ -101,12 +133,16 @@ describe('withVariables function', () => {
   });
 
   it('returns an attachment object when passed multiple magic variables in a string', () => {
-    const uuid1 = 'b74c81a8-192a-463f-a0a6-2d327963714f';
-    const uuid2 = '4e8b6858-cf85-4ffe-9019-efb421248510';
-    const uuid3 = 'af9c2e97-8af2-4029-8664-18fb158dbd16';
-    const uuid4 = '10407a89-ef91-4eb0-a3ac-2ea62a8c25b6';
-
-    const actual = withVariables`${uuid1} Going ${uuid2}${uuid3} to the ${uuid4} blacksmith`;
+    const magic1 = actionOutput();
+    magic1.OutputUUID = 'b74c81a8-192a-463f-a0a6-2d327963714f';
```

Comment:
Instead of overriding the `OutputUUID` here, could we instead just use `magic.OutputUUID` in the `expected` object, as you've done [here](https://github.com/joshfarrant/shortcuts-js/pull/14/files#diff-3cb72db0b0eb735182c79bade216f24aR24)? Your method is much nicer than my previous method of setting the UUIDs manually.